web/js/pages/list_rrd.js

```diff
@@ -57,11 +57,40 @@ function formatLabel(date, period) {
 }
 
 function getChartOptions() {
+	const currentTheme = getCurrentTheme();
+	const textColor = currentTheme === 'dark' ? '#cdcdcd' : '#7c7c7c';
```

Comment:
If users create his own dark theme this check will probably not work...

How ever I don' t know if there is a easy fix for it..

Reply:
Good point. I'll try to use CSS variables for this instead, that can be more easily overridden.